guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
         ├── Foo.jsx             React component
         ├── Foo.test.jsx        React component tests
         └── Foo.example.jsx     React component used to render an example
+        └── Foo.example.html    HTML example
 ```
 
-# CSS components
+## CSS components
 
 [KSS](https://github.com/kss-node/kss-node) is the primary way documentation is written. They establish what pages get created and the content on those pages. We use KSS to parse CSS comment blocks to extract documentation, and it expects your comment block to follow this particular format:
 
-```css
+```scss
 /*
 [Heading]
 
 [Description]
 
-[@flags]
-
-Markup:
-[Inline HTML/EJS or filename]
+[Flags]
 
 [Modifiers]
 
+Markup: [filename]
```

Comment:
the way the system builds now, all the markup is inline. I'd keep this as it was unless we intend to rewrite all our files to follow filename-only conventions.

Reply:
I was thinking it would be easier to validate and view as a separate file with just the HTML instead of inlining the markup. Maybe this could be some follow on work to extract the inline code to separate files?